From 51d0011cdec1b94341b59a9607f174970c3dc817 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Thu, 14 Nov 2013 19:42:18 +0000
Subject: [PATCH 128/462] Mantis 29552: device documentation

ReleaseNotes.schema.json; V1.0.0
initial version for structured release notes using JSON schema file

added enumeration for NodeContext

git-svn-id: svn://localhost/SelfServiceBuild/trunk@190 2eb42a87-3827-0410-b767-dc15055982e4
---
 .../Build/releasenotes/src/ReleaseNotes.schema.json        | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/releasenotes/src/ReleaseNotes.schema.json b/SelfServiceBuild/Build/releasenotes/src/ReleaseNotes.schema.json
index 5a9b7d5..ebdf5db 100644
--- a/SelfServiceBuild/Build/releasenotes/src/ReleaseNotes.schema.json
+++ b/SelfServiceBuild/Build/releasenotes/src/ReleaseNotes.schema.json
@@ -167,7 +167,7 @@
       },
       "NodeContext": {
        "type": "array",
-       "items": {"type": "string"}
+       "items": {"$ref": "#/definitions/NodeContext"}
       },
       "NotFoundHandling": {
        "type": "array",
@@ -339,6 +339,11 @@
    "enum": ["Bugfix", "Extension", "Change", "Initial"],
    "type": "string"
   },
+  "NodeContext": {
+    "enum": ["Unknown", "GUI-Flash", "GUI-Html", "GUI-Kiosk", "BL", "Operating", "OS", "LH", "Location",
+             "Confidential", "Hardware", "Application", "Service", "User-Interface", "Path"],
+    "type": "string"
+  },
   "ActionType": {
    "enum": ["Add", "Change", "Remove"],
    "type": "string"
-- 
2.41.0.windows.1

